crates/re_ui/src/list_item2/scope.rs

```diff
@@ -125,6 +182,21 @@ pub fn list_item_scope<R>(
         ui.clip_rect().x_range()
     };
 
+    // Set up the state for this scope.
+    state.left_x = ui.max_rect().left();
+    state.left_column_width = if state.max_desired_left_column_width > 0.0 {
+        Some(
+            // TODO(ab): this heuristics can certainly be improved, to be done with more hindsight
+            // from real-world usage.
+            state
+                .max_desired_left_column_width
+                .at_most(0.7 * ui.max_rect().width()),
```

Comment:
I think there should be a minimum width we don't go below (e.g. 32 points), and force the parent scope to have scroll bars instead

```suggestion
                .at_most(0.7 * ui.max_rect().width()),
                .at_least(MIN_LEFT_COLUMN_WIDTH)
```

Reply:
I agree in principle, but suggest postponing any complexity-inducing improvement to when we actually reach into real world issues. I find very hard to predict how things will break exactly.